tools/manifest/item.py

```diff
@@ -84,7 +84,15 @@ def id(self):
 
     @property
     def url(self):
-        return urljoin(self.url_base, self._url)
+        # we can outperform urljoin, because we know we just have path relative URLs
+        if self._url.startswith("/"):
```

Comment:
Is `self.url[0]` not going to be faster?

Reply:
oh, probably, hopefully we never have empty strings!